sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -46,7 +46,7 @@
     from azure.core.credentials import TokenCredential
 
 _LOGGER = logging.getLogger(__name__)
-_Address = collections.namedtuple("Address", "hostname path")
+_Address = collections.namedtuple("Address", "hostname path")   # type: ignore
```

Comment:
@yunhaoling - Can we change the inner named tuple to `_Address`?

Reply:
yes, I think we could rename the tuple as long as this namedtuple is kept for internal usage.

Reply:
yep only used internally, not exposed to user.